### PR TITLE
Iss#59 Added script to move and take photo at specific location.

### PR DESCRIPTION
**Pull request by Hsenrab**
This review should also look at the camera functionality in general both debug and real. Note that the efficiency of the real camera should not be reviewed as there is another issue in for that.

### CODE REVIEW

File: scripts/take_photo.py [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+import context
+import roboplot.config as config
+import roboplot.core.curves as curves
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+from roboplot.core.camera.camera_wrapper import Camera
+
+try:
+    # Commandline arguments
+    parser = argparse.ArgumentParser(description='Draw a circle.')
```
Comment (JackBuck) [inline scripts/take_photo.py]: Possibly you wanted to change this?

File: scripts/take_photo.py [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+import time
+
+import numpy as np
+
+import context
+import roboplot.config as config
+import roboplot.core.curves as curves
+import roboplot.core.hardware as hardware
+from roboplot.core.gpio.gpio_wrapper import GPIO
+from roboplot.core.camera.camera_wrapper import Camera
```
Comment (JackBuck) [inline scripts/take_photo.py]: At least on this branch, this should be
`from roboplot.core.Camera.camera_wrapper import Camera `
(i.e. capital 'C').
But I know you've changed it elsewhere.

File: scripts/take_photo.py (hunk 1 above)
Comment (Hsenrab) [inline scripts/take_photo.py]: Changed below will wait to merge until you have tested it on the pi. Please can you commit any tweaks you make to get it working on the pi.

**Comment by JackBuck** [comment]
@Hsenrab I got confused with github then and thought I would have another chance to add review comments.

I know that the `pi_camera` saves a photo as 'Photo_0.jpg' when I run this script, but it is a bit buried! Could you put in a `filepath` argument to this script to allow me to specify a save location? (I don't care if it saves the debug image in addition!)

**Pull request merged**